### ISSUE TITLE
Circuit Design

### ISSUE THREAD

**Issue by ReneAlimisi** [ecraft2learn/uui#70]
When click on the Circuit Design...the following message appears: 

Oops!
403 - Access forbidden
Get me out of here

<img width="477" alt="circuitdesign" src="https://user-images.githubusercontent.com/46059341/50367595-06401700-058a-11e9-9916-243a79b90617.png">

**Comment by heijler** [comment]
I get the same issue also on Circuit design as well as on Tinkercad 3D Design. 558de1f4a1feaa42d5e7aaed8a1cbac388b701e9 related @ToonTalk ?

**Comment by ToonTalk** [comment]
If not logged into to TinkerCad I see the same problem. If logged in then it works fine. It used to be that if you weren't logged in it would take you to a log in page but apparently AutoDesk changed things.

I'm investigating but so should others that I added as "assignees".

**Comment by ToonTalk** [comment]
I tried a few different ways to catch this error and inform the user that they need to log in to TinkerCad but all failed. We could produce a warning on every use instead of what would be ideal which is the warning is shown only if the page loads with an error. What do people think we should do?

**Comment by Calkin-eC2L** [comment]
Ken, just go ahead with a solution. If possible, on click to Tinkercad (circuits or 3D modelling) a pop-up window in the UUI shows a warning: log in to Tinkercad first to avoid errors. 
If this is not possible, then the way that you describe should be implemented. This needs to work over this weekend...

**Comment by ToonTalk** [comment]
I agree the warning is better than the current situation so I just commited it. Should be deployed within an hour.

Leaving this open in case someone has an idea how to produce the warning only if not logged into TinkerCad.

**Comment by Calkin-eC2L** [comment]
I see the popup window when clicking the 1st time on tinkercad. This needs to happen also when clicking to circuits. **would it be better to make it so that the warning appears always when tinkercad is clicked?** at the moment it only appears once (since it is not checking whether login has occurred)

![image](https://user-images.githubusercontent.com/35801506/50538060-ee602880-0b71-11e9-9a8a-0509cab4ebc9.png)

**Comment by ToonTalk** [comment]
While I don't feel strongly about re-opening this I think showing the warning once is the best UI practice. If a user is logged into TinkerCad they will get this warning every time they launch one of the TinkerCad tools. Pretty annoying. What I did was make it remember that it has already displayed the warning until the UUI tab has been closed or refreshed.

Again maybe not worth re-opening.
